library/core/src/ascii/ascii_char.rs

```diff
@@ -567,9 +567,10 @@ impl fmt::Display for AsciiChar {
 #[unstable(feature = "ascii_char", issue = "110998")]
 impl fmt::Debug for AsciiChar {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        #[inline]
```

Comment:
Any reason not to keep this `#[inline]`?

Reply:
As far as I understand, it doesn’t do anything. inline annotation matters on cross-crate boundaries only.